libse/SubtitleFormats/PhoenixSubtitle.cs

```diff
@@ -27,33 +27,43 @@ public class PhoenixSubtitle : SubtitleFormat
 
         public override bool IsMine(List<string> lines, string fileName)
         {
-            if (fileName?.EndsWith(".pjs", StringComparison.OrdinalIgnoreCase) == false)
```

Comment:
This was preventing detecting when there is no file supplied which mostly happen when the format is changed from Format combobox in main-window